packages/components/src/custom-select-control-v2/test/index.tsx

```diff
@@ -12,7 +12,7 @@ import { useState } from '@wordpress/element';
 /**
  * Internal dependencies
  */
-import { CustomSelect as UncontrolledCustomSelect, CustomSelectItem } from '..';
+import UncontrolledCustomSelectControl from '..';
```

Comment:
Should we use `UncontrolledCustomSelectControlV2` instead?

Reply:
Addressed in bf056b6 👍